### DISCUSSION TITLE
Questions/Thoughts Encountered on Sensitivity Analysis Task

### DISCUSSION THREAD

**Discussion by shelbysawyer** [Riverscapes/riverscapes-tools#736] [General]
As Lauren and I work through performing sensitivity analyses for GNAT and VBET, we are running into some questions. The answers to these will not only be useful for us now, but also for documentation purposes down the road. We will add additional questions as they arise.

1. How do IGO metrics get calculated when the window search distance extends beyond the length of the channel? An example of this would be an IGO that is 200 m down from the headwater of a medium channel, and the window size is 800 m (400 up, 400 down).

2. Right now, we are just using the word "window" to describe the distance/area of DGOs that get aggregated into IGOs. I think this is confusing for users. For example, right now, we are using the term "200 m window" to describe an IGO that looks 100 m up and 100 m down. This gives the impression that the window itself is 200 m, but in reality, the analysis itself is being done for a 300 m length. I think it might be useful to consider using the terms "search window" and "analysis window", or pick one and be very clear to users about what it means. Please check out the illustration below:
![window](https://user-images.githubusercontent.com/73311751/230215414-3330a3ad-3135-43ca-b998-053fa7e85863.JPG)

**Comment by lauren-herbine** [comment]
I very much agree- I think this is also getting confused within our group, not just for theoretical future users. I am realizing that I have not been sure if we've been referring to the analysis window or the "search window" in past conversations.
@joewheaton @KellyMWhitehead @philipbaileynar @jtgilbert if you all agree on this new nomenclature, it would be useful moving forward to use these distinctions.
